src/Symfony/Component/HtmlSanitizer/HtmlSanitizerConfig.php

```diff
@@ -0,0 +1,469 @@
+     * Passing "*" will allow all currently allowed elements to use this attribute.
+     */
+    public function allowAttribute(string $attribute, array|string $allowedElements): static
+    {
+        $clone = clone $this;
+        $allowedElements = ('*' === $allowedElements) ? array_keys($clone->allowedElements) : (array) $allowedElements;
+
+        // For each configured element ...
+        foreach ($clone->allowedElements as $element => $attrs) {
+            if (\in_array($element, $allowedElements, true)) {
+                // ... if the attribute should be allowed, add it
+                $clone->allowedElements[$element][$attribute] = true;
+            } else {
+                // ... if the attribute should not be allowed, remove it
+                unset($clone->allowedElements[$element][$attribute]);
+            }
+        }
+
+        return $clone;
+    }
+
+    /**
+     * Configures the given attribute as dropped.
+     *
+     * Dropped attributes are attributes the sanitizer should remove from the input.
+     *
+     * A list of elements on which to drop this attribute can be passed as a second argument.
+     * Passing "*" will drop this attribute from all currently allowed elements.
+     *
+     * Note: when using an empty configuration, all unknown attributes are dropped
+     * automatically. This method let you drop attributes that were allowed earlier
+     * in the configuration.
+     */
+    public function dropAttribute(string $attribute, array|string $droppedElements): static
+    {
+        $clone = clone $this;
+        $droppedElements = ('*' === $droppedElements) ? array_keys($clone->allowedElements) : (array) $droppedElements;
+
+        foreach ($droppedElements as $element) {
+            if (isset($clone->allowedElements[$element][$attribute])) {
+                unset($clone->allowedElements[$element][$attribute]);
+            }
+        }
+
+        return $clone;
+    }
+
+    /**
+     * Forcefully set the value of a given attribute on a given element.
+     *
+     * The attribute will be created on the nodes if it didn't exist.
+     */
+    public function forceAttribute(string $element, string $attribute, string $value): static
+    {
+        $clone = clone $this;
+        $clone->forcedAttributes[$element][$attribute] = $value;
+
+        return $clone;
+    }
+
+    /**
+     * Registers a custom attribute sanitizer.
+     */
+    public function withAttributeSanitizer(AttributeSanitizerInterface $sanitizer): static
+    {
+        $clone = clone $this;
+        $clone->attributeSanitizers[] = $sanitizer;
+
+        return $clone;
+    }
+
+    /**
+     * Unregisters a custom attribute sanitizer.
+     */
+    public function withoutAttributeSanitizer(AttributeSanitizerInterface $sanitizer): static
+    {
+        $clone = clone $this;
+        unset($clone->attributeSanitizers[array_search($sanitizer, $clone->attributeSanitizers, true)]);
```

Comment:
```suggestion
        $clone->attributeSanitizers = array_values(array_filter(
            $this->attributeSanitizers,
            static fn ($current) => $current !== $sanitizer
        ));
```

Nit-picking: This way we make sure, the array is a list again after removing the item.